.github/workflows/build-json.yml

```diff
@@ -103,6 +111,15 @@ jobs:
               with:
                 name: cldr-json
                 path: ./cldr-json/dist
+            - name: Push changes
+              if: ${{ github.event.inputs.push-changes }}
+              working-directory: cldr-json
+              run: |
+                git config --global user.name "GitHub Actions"
+                git config --global user.email "actions@github.com"
+                git add cldr-json
+                git commit --verbose -F - < (echo "Automated update; echo; cat ../cldr-identity.txt")
```

Comment:
this line isn't quite right, too many levels of quotes. it will result in a commit message `Automated update; echo; cat..`

I think the first line could give a little more details as well.

Probably instead something like

`( echo  ... ) | git commit ... -F -`